src/fpu/fpu.cpp

```diff
@@ -152,8 +152,8 @@ void FPU_ESC0_Normal(Bitu rm) {
 
 void FPU_ESC1_EA(Bitu rm,PhysPt addr) {
 // floats
-	Bitu group=(rm >> 3) & 7;
-	Bitu sub=(rm & 7);
+    const uint8_t group=(rm >> 3) & 7;
+    const uint8_t sub=(rm & 7);
```

Comment:
indentation (just reformat the whole file)

Reply:
yeah, seems like the script doesn't quite do the right thing all the time.

Reply:
oh, wait, looks like you were looking at an outdated commit, nevermind